packages/devtools_app/lib/src/screens/debugger/variables.dart

```diff
@@ -267,17 +261,12 @@ class _TextSelectionControlsToolbarState
       widget.clipboardStatus.addListener(_onChangedClipboardStatus);
       oldWidget.clipboardStatus.removeListener(_onChangedClipboardStatus);
     }
-    widget.clipboardStatus.update();
   }
 
   @override
   void dispose() {
     super.dispose();
-    // When used in an Overlay, it can happen that this is disposed after its
-    // creator has already disposed _clipboardStatus.
-    if (!widget.clipboardStatus.disposed) {
-      widget.clipboardStatus.removeListener(_onChangedClipboardStatus);
-    }
+    widget.clipboardStatus.removeListener(_onChangedClipboardStatus);
```

Comment:
why is the use after dispose case now resolved?

Reply:
This pr solves it https://github.com/flutter/flutter/commit/5f3bee55ad4804984256dc6bfd762fd0911db901